src/core/thread/mesh_forwarder_ftd.cpp

```diff
@@ -824,7 +824,11 @@ void MeshForwarder::GetForwardFramePriority(RxInfo &aRxInfo, Message::Priority &
     error = GetFramePriority(aRxInfo, aPriority);
 
 exit:
-    if (error != kErrorNone)
+    if (error == kErrorNotFound)
+    {
+        LogInfo("No frame priority, using normal priority, %s", aRxInfo.ToString().AsCString());
+    }
+    else if (error != kErrorNone)
```

Comment:
Thanks @agners. This change as is is fine. Alternatively, we can consider changing the `LogNote` below to `LogInfo`, or maybe even removing it fully, which may be simpler. Thoughts?

Reply:
If we expect `GetFramePriority()` to regularly return an error, I suggest removing the log message.

Reply:
Yes. I would also prefer to remove the log.

Reply:
Removing or maybe alternatively lower to debug by default, both are fine by me.

Out of interest, why don't we have priority in some cases? :thinking: